src/Components/App/App.js

```diff
@@ -43,7 +45,8 @@ export default class App extends React.Component {
     }
 
     authenticate(provider) {
-        console.log(`Trying to log in with ${provider}`);
+        console.log(`Trying to log in with ${provider}`); 
+        this.setState({userLoading: true})
         base.authWithOAuthRedirect(provider, this.authHandler);
```

Comment:
should we change this to 
`base.authWithOAuthPopup(provider, this.authHandler);`
?